lib/instance.js

```diff
@@ -116,6 +116,30 @@ class Instance {
         this._cliConfig.set('node-version', value).save();
     }
 
+    /**
+     * Node Binary path used when installing this version
+     *
+     * @property nodeBinary
+     * @type string
+     * @public
+     */
+    get nodeBinary() {
+        return this._cliConfig.get('node-binary', process.argv[0]);
```

Comment:
would it make sense to use process.execPath instead?

Reply:
Yep, it looks like `process.execPath === process.argv[0]`